var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -32,9 +32,14 @@ class PyScipy(Package):
     version('0.17.0', '5ff2971e1ce90e762c59d2cd84837224')
     version('0.15.1', 'be56cd8e60591d6332aac792a5880110')
     version('0.15.0', '639112f077f0aeb6d80718dc5019dc7a')
+    
+    variant('atlas', default=False,  description='Compile with ATLAS')
 
     extends('python')
     depends_on('py-nose', type='build')
+    depends_on('atlas',   when='+atlas')
+    depends_on('blas',    when='~atlas')
```

Comment:
it did work though because blas/lapack were brought from `py-numpy+blas+lapack` which need them.
But i think it's good to add them hear as well for clarity.